charts/kubermatic/values.yaml

```diff
@@ -45,8 +45,6 @@ kubermatic:
   # clusters on one central Service of type LoadBalancer via the NodePort proxy or "LoadBalancer" to create a LoadBalancer service per cluster
   # **Note:** The `seed_dns_overwrite` setting of the `datacenters.yaml` doesn't have any effect if this is set to `LoadBalancer`
   exposeStrategy: "NodePort"
-  # base64 encoded presets.yaml. Predefined presets for all supported providers.
```

Comment:
Please undo all changes to the old `kubermatic` Helm chart. The chart is not used anymore and will just be removed alltogether in the near future. Until then, just forget it even exists. ^^

Reply:
Reverted all changes related to helm chart.